serving/source-sink/build.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
+    tonic_build::configure()
+        .build_server(true)
+        .compile(
+            &[
+                "proto/source.proto",
```

Comment:
What is the reason the client needs to keep the proto files?

Reply:
we do not need to, we can copy at build time. this was the reason for #1910

Reply:
we will handle this in follow up PR.